src/utils.jl

```diff
@@ -155,7 +145,7 @@ More convenient macro expansion, e.g.
     @expand @time foo()
 """
 macro expand(ex)
-  :(alias_gensyms(macroexpand($(ex,)[1])))
+  :(alias_gensyms(macroexpand($(@__MODULE__), $(ex,)[1])))
```

Comment:
I think this is what @vtjnash suggested in the original issue – I'm not up to date on all the macro changes myself though.